src/Search_Replace_Command.php

```diff
@@ -518,6 +522,7 @@ private function php_handle_col( $col, $primary_keys, $table, $old, $new ) {
 		$col_sql          = self::esc_sql_ident( $col );
 		$where            = $this->regex ? '' : " WHERE $col_sql" . $wpdb->prepare( ' LIKE BINARY %s', '%' . self::esc_like( $old ) . '%' );
 		$primary_keys_sql = implode( ',', self::esc_sql_ident( $primary_keys ) );
+		// phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared -- escaped through self::esc_sql_ident
 		$rows             = $wpdb->get_results( "SELECT {$primary_keys_sql} FROM {$table_sql} {$where}" );
```

Comment:
As there is now a comment breaking the assignment list, the whitespace before the `=` here needs to be reduced to one space.